nginx-php5/Dockerfile

```diff
@@ -3,15 +3,14 @@ FROM unit9/base:latest
 MAINTAINER Artiom Vasiliev <artiom@unit9.com>
 
 COPY ["php-fpm.rc", "/etc/service/php-fpm/run"]
+COPY ["nginx.rc", "/etc/service/nginx/run"]
 
 RUN apt-get update && \
     apt-get install -y nginx php5 php5-common php5-curl php5-fpm php5-gd \
         php5-mysql php5-xmlrpc php5-xcache php5-memcache && \
     rm -rf /var/cache/apt /var/lib/apt/lists && \
-    chmod +x /etc/service/php-fpm/run
+    chmod +x /etc/service/php-fpm/run /etc/service/nginx/run
```

Comment:
You don't need an explicit `chmod +x` if the files were already `+x`'d in the git source repository.